src/xopen/__init__.py

```diff
@@ -441,6 +443,32 @@ def __init__(
         super().__init__(path, ["pigz"], mode, compresslevel, "-p", threads)
 
 
+class PipedPBzip2Reader(PipedCompressionReader):
+    """
+    Open a pipe to pbzip2 for reading a bzipped file.
+    """
+    def __init__(self, path, mode: str = "r", threads: Optional[int] = None):
+        super().__init__(path, ["pbzip2"], mode, "-p", threads)
+
+class PipedPBzip2Writer(PipedCompressionWriter):
+    """
+    Write bzip2-compressed files by running an external pbzip2 process and
+    piping into it. pbzip2 can compress using multiple cores.
+    """
+
+    def __init__(
+        self,
+        path,
+        mode: str = "wt",
+        compresslevel: Optional[int] = None,
+        threads: Optional[int] = None,
+    ):
+        """
+        """
```

Comment:
Please remove the empty docstring (or write something)

Reply:
Done!